<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kukulatorClass</class>
 <widget class="QMainWindow" name="kukulatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>kukulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="b_dec">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>50</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: 00FF00;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>DEC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_bin">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>50</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: 00FF00;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>BIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_hex">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>50</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: 00FF00;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>HEX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_plus">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_multiply">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_dot">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_minus">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_devide">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_equals">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
    <property name="cursorPosition">
     <number>3</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_hex">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>200</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="cursorPosition">
     <number>1</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_bin">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>200</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="cursorPosition">
     <number>1</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dec">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>200</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="cursorPosition">
     <number>1</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="b_clear">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}
QPushButton:pressed{
	background-color: A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="kukulator.qrc"/>
 </resources>
 <connections/>
</ui>
